arwen/common.go

```diff
@@ -1,14 +1,15 @@
 package arwen
 
 import (
-	"github.com/ElrondNetwork/wasm-vm/config"
 	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/ElrondNetwork/wasm-vm/config"
+	"github.com/ElrondNetwork/wasm-vm/executor"
 )
 
 // ArwenVersion returns the current arwen version
 const ArwenVersion = "v1.4"
 
-// Size in bytes of a WASM Linear Memory Page 
+// Size in bytes of a WASM Linear Memory Page
```

Comment:
This was an older mistake on `master`, fixable with the line below, if it's alright to add it in this PR:

```suggestion
// WASMPageSize is the size in bytes of a WASM Linear Memory Page
```